app/controllers/repositories_controller.rb

```diff
@@ -27,7 +27,7 @@ class RepositoriesController < ApplicationController
   def index
     respond_to do |format|
       format.html do
-        render 'empty_index' if @repositories.blank?
+        render 'empty_index' if Repository.accessible_by_teams(current_team).blank?
```

Comment:
ins't '.none?' a bit faster?

Reply:
Hm.. ya, could be, but I was too slow for @radiokills